build/squire-raw.js

```diff
@@ -2254,17 +2254,33 @@ var onDrop = function ( event ) {
     while ( l-- ) {
         switch ( types[l] ) {
         case 'text/plain':
+        case 'Text': // IE Specific
             hasPlain = true;
             break;
         case 'text/html':
             hasHTML = true;
             break;
         default:
-            return;
+            break;
         }
```

Comment:
The way it was written, it would return from the function the first time it found a type that was not text or html. Really it needs to check the entire list of types to see if any are text or html.